project/Dependencies.scala

```diff
@@ -72,7 +72,7 @@ object Dependencies {
   val playJsonExtensions = "ai.x" %% "play-json-extensions" % playJsonExtensionsVersion
   val playJsonJoda = "com.typesafe.play" %% "play-json-joda" % playJsonVersion
   val playIteratees = "com.typesafe.play" %% "play-iteratees" % "2.6.1"
-  val atomRenderer = "com.gu" %% "atom-renderer" % "1.1.6"
+  val atomRenderer = "com.gu" %% "atom-renderer" % "1.2.0"
```

Comment:
versions mismatch between scala and js?

Reply:
Yeah unfortunately I realised I had to make this extra change on the client side: https://github.com/guardian/atom-renderer/pull/85